am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/DefaultRolePermissions.java

```diff
@@ -12,6 +13,7 @@
 @Data
 @Builder
 @AllArgsConstructor
+@NoArgsConstructor //required for JBI Bean Mapper RolePermissionsForCaseTypeEnvelopeReducer
```

Comment:
comment to be corrected - "required for J**D**BI Bean Mapper RolePermissionsForCaseTypeEnvelopeReducer"

Reply:
done